packages/maker.js/src/core/dxf.ts

```diff
@@ -147,21 +147,20 @@ namespace MakerJs.exporter {
             return polylineEntity;
         }
 
-        function mtext(caption: ICaption) {
+        function text(caption: ICaption) {
             const center = point.middle(caption.anchor);
-            const mtextEntity: DxfParser.EntityMTEXT = {
-                type: "MTEXT",
-                position: {
-                    x: round(center[0], opts.accuracy),
-                    y: round(center[1], opts.accuracy)
-                },
-                height: opts.fontSize,
+            const textEntity: DxfParser.EntityTEXT = {
+                type: "TEXT",
+                startPoint: appendVertex(center, null),
+                endPoint: appendVertex(center, null),
+                layer: defaultLayer(null, null),
```

Comment:
@danmarshall - is there a way to get the model's layer here?  Right now it will just put it on layer `0`.

Reply:
Hi @boomer196 - looks like we'd need to refactor this a bit to have the layer info. Maybe I will send you a PR.

Reply:
Whoops - I meant to send this as a PR but I accidentally checked into your branch. See my commit at https://github.com/boomer196/maker.js/commit/8e276e68f36817c7ea45343559929126d56a57a3

Reply:
No worries.  I'll work with what you submitted and get the CI passing.  I would still like to look into being able to specify a layer that is different from the model, but let's worry about that after this PR gets merged.  This PR fixes my immediate need.

Reply:
Right. It's always easy enough to create an empty model with only a layer and a caption.